src/R/Interpreters/Impl/IMicrosoftRClientInstaller.cs

```diff
@@ -10,6 +11,6 @@ namespace Microsoft.R.Interpreters {
     /// of the Microsoft R Client. Exported via MEF.
     /// </summary>
     public interface IMicrosoftRClientInstaller {
-        void LaunchRClientSetup(ICoreShell coreShell, IFileDownloader downloader = null);
+        void LaunchRClientSetup(IServiceContainer coreShell, IFileDownloader downloader = null);
```

Comment:
This is xplat already. If you mean the part that is in the `Microsoft.R.Interpreters.Windows`, broker either need to load it dynamically or reference directly, but in both cases I'd rather leave it in a small separate assembly rather than make it part of a `Microsoft.VisualStudio.R.Package`.